### DISCUSSION TITLE
https://lucashmalcantara.github.io/k/Lucas-Alc%C3%A2ntara/About-Me

### DISCUSSION THREAD

**Discussion by giscus** [lucashmalcantara/k#1] [Announcements]
# https://lucashmalcantara.github.io/k/Lucas-Alc%C3%A2ntara/About-Me

My name is Lucas Alcântara. I’m from Brazil. I have a Bachelor’s degree in Information Systems (PUC Minas). Here are my social media accounts: GitHub LinkedIn .

https://lucashmalcantara.github.io/k/Lucas-Alc%C3%A2ntara/About-Me

**Comment by lucashmalcantara** [comment]
Oi!
